src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import Avatar from './Avatar';
+import { UserProfile } from 'services/user';
+
+import styles from './Layout.module.css';
+
+interface ComponentProps {
+  show: boolean;
+  setShow: Dispatch<SetStateAction<boolean>>;
+  user: UserProfile;
+}
+
+const Sidebar = ({ show, setShow, user }: ComponentProps) => {
+  return (
+    <>
+      {show && (
+        <div
+          className="fixed top-0 right-0 h-screen w-screen"
+          data-testid="sidebar-overlay"
+          onClick={() => setShow(false)}
+        />
+      )}
+      <aside className={styles.sidebar} aria-hidden={!show}>
+        <section className="flex justify-between items-center">
+          <div className="font-extrabold text-2xl text-white">{user.email}</div>
```

Comment:
Maybe set the `max-width` for this `div` to avoid the sidebar breaking when the email is long
![Screen Shot 2565-05-12 at 10 16 09](https://user-images.githubusercontent.com/1772999/167985188-a3e471e5-a14e-4796-9768-172976999d21.png)

Reply:
Updated in [95c8bb2](https://github.com/hanam1ni/react-surveyor/pull/60/commits/95c8bb2aab315c475ed8440499f07b711892a192) 
<img width="445" alt="Screen Shot 2565-05-12 at 13 45 31" src="https://user-images.githubusercontent.com/14077479/168009505-c2542dd6-d8cf-4b6f-96e9-7e5428e97cc3.png">